BlueprintUICommonControls/Sources/Label.swift

```diff
@@ -45,3 +68,27 @@ public struct Label: Element {
     }
 
 }
+
+extension Label {
+    public enum LineHeight {
+        public enum Alignment {
+            /// Align text to the top of the available line height, with extra space added at the bottom.
+            /// This makes line height behave like traditional leading.
+            case top
+
+            /// Center text within the available line height. This makes line height behave like half-leading,
+            /// and matches the model used by CSS.
+            case center
+
+            /// Align text to the bottom of the available line height, with extra space added at the top.
+            /// This is the default behavior of `UILabel` on iOS.
+            case bottom
+        }
+
+        /// Use the default line height of the label's font.
+        case font
```

Comment:
``default``?

Reply:
This is more explicit about what this case is, though? `default` doesn't tell you anything about the actual line height being used.

Reply:
Seeing `.font` inline seems more confusing to me than `.default`, imo, given iOS SDKs usually use `default` to refer to what the system would usually do, but this is also fine.